### PR TITLE
Update branding and remove dead code 

### PR DESCRIPTION
**Pull request by stvstnfrd**
The `README` still referenced the old repository.

The unimplemented feature, I've wanted to kill that code for ages, but am just now making it happen.

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@caesar2164 Please review.

**Comment by stvstnfrd** [comment]
@caesar2164 I've tacked on a few more commits. Each commit has additional details in the message. In short, they remove superfluous code and correct some issues.

I have a larger refactoring branch in-progress; I actually cherry-picked a few of them here. That one will be much larger, so I want to get some of the smaller bits in first.

File: gen_cert.py [hunk 1]
```diff
@@ -258,10 +258,6 @@ def interstitial_factory():
         self.cert_label_plural = cert_data.get('CERTS_ARE_CALLED_PLURAL', CERTS_ARE_CALLED_PLURAL)
         self.course_association_text = cert_data.get('COURSE_ASSOCIATION_TEXT', 'a course of study')
 
-    def delete_certificate(self, delete_download_uuid, delete_verify_uuid):
```
Comment (caesar2164) [inline gen_cert.py:261]: Yay for removing cruft!

**Comment by caesar2164** [comment]
This PR looks GREAT thus far! :+1:

**Comment by stvstnfrd** [comment]
@caesar2164 I'm ready to merge this as-is; I'll submit a follow-up PR for additional work.

**Comment by caesar2164** [comment]
go for it!

**Pull request merged**